vignettes/subsetting_tables.Rmd

```diff
@@ -168,3 +168,36 @@ Note the return value of `cell_values` is always a list even if you specify a pa
 ```{r}
 cell_values(tbl2, c("RACE", "ASIAN", "AGE",  "Mean"), c("ARM", "A: Drug X", "SEX", "F"))
 ```
+
+## Rows and cell values alignments
+
+It is possible to align the content of both rows (`in_rows`) and cell values (`rcell`) by assigning `"left"`, `"center"` (default), and `"right"` to `.aligns` and `align` arguments, respectively.
+
```

Comment:
`.aligns` actually assigns alignment at the cell level, not the row level (the same as `.formats` does). Please correct  this in the vignette and explicitly document it in the documentation to `in_rows` as well.

(This is somewhat unintuitive but we can see that its the case from the relevant code within in_rows:

```
<snip>
        l2 <- mapply(rcell, x = l, format = .formats, footnotes = .cell_footnotes %||% 
            list(NULL), align = .aligns, format_na_str = .format_na_strs %||% 
            list(NULL), SIMPLIFY = FALSE)
<snip>
```

Reply:
After a few trials, I think I understood the machinery. Here is an example for the above with alignment (it is in the examples):
```r
basic_table() %>%
  split_cols_by("ARM") %>%
  split_rows_by("SEX", split_fun = drop_split_levels) %>%
  analyze(c("AGE"), function(x) {
    in_rows(
      "mean" = rcell(mean(x), align = "right"),
      "sd" = rcell(sd(x), align = "left"), .formats = c("xx.x")
    )
  }, show_labels = "visible", na_str = "NE") %>%
  build_table(ex_adsl)
```

oputputs:
```
                   A: Drug X   B: Placebo   C: Combination
——————————————————————————————————————————————————————————
F                                                         
  AGE                                                     
    mean                32.8         34.1             35.2
    sd             6.1         7.1          7.4           
M                                                         
  AGE                                                     
    mean                35.6         37.4             35.4
    sd             7.1         8.7          8.2           
U                                                         
  AGE                                                     
    mean                31.7         31.0             35.2
    sd             3.2         5.7          3.1           
UNDIFFERENTIATED                                          
  AGE                                                     
    mean                28.0           NE             45.0
    sd             NE          NE           1.4           
```